qualang_tools/config/__init__.py

```diff
@@ -3,10 +3,13 @@
     compress_integration_weights,
     plot_integration_weights,
 )
+from qualang_tools.config.waveform_tools import (

```

Comment:
what is the disadvantage to import *, vs only import the exact waveforms to use? is there any memory or delay issue? -- trying to learn here

Reply:
It's considered better practice in general (If you import things you don't need, you waste memory).
I don't really agree with that when writing scripts, but for packages, I follow that.